Specs/Flatglass.md

```diff
@@ -0,0 +1,90 @@
+# umati showcase information model for Flat Glass Processing
+
+In the following the umati showcase describes the additional information on top of the [OPC 40301 UA for Flat Glass Processing](https://www.vdma.org/viewer/-/v2article/render/15410388) companion specification.
+
+Please refer to the provided xml-Nodesets or UaModeler-project for exemplary implementation.
+
+The following describes the umati showcase information model for Flat Glass Processing.
+
+For participants that are familiar with the information model, we have included comments and descriptions about changes between the release candidate information model and the showcase information model.
+
+All participants not familiar with the release candidate information model do not need to review these changes in detail but rather follow the described model and refer to the provided xml Nodeset files and UA modeler project.
+
+## NodeSets for the showcase
+
+[Normative NodeSet hosted by the OPC Foundation](https://github.com/OPCFoundation/UA-Nodeset/tree/v1.04/Glass/Flat)
```

Comment:
Change link: https://github.com/OPCFoundation/UA-Nodeset/tree/latest/Glass/Flat

Reply:
Changed it.